### ISSUE TITLE
Add a button to preview markdown

### ISSUE THREAD

**Issue by Freeplayg** [userstyles-world/userstyles.world#53]
**Describe the change you'd like**:
Add a button below text boxes that support markdown to preview it, so you will know how it will look before making it public.
Also maybe a button link to a markdown guide would be helpful

**Comment by vednoc** [comment]
@Gusted I wanna know if you have any ideas how to implement this differently. ATM I'm thinking of adding another API endpoint for rendering Markdown then using fetch API to send/receive the data.

**Comment by Gusted** [comment]
Yeah, I guess so. I thought about something similar. Maybe If Il get around I can implement it later today, but OAuth is demanding time.

**Comment by vednoc** [comment]
I could work on this; was just curious if you had something else in mind.

**Comment by Gusted** [comment]
> I could work on this; was just curious if you had something else in mind.

Super epic WASM file that does the processing 😄
